src/main/java/frc/robot/RobotContainer.java

```diff
@@ -186,7 +179,7 @@ private void configureButtonBindings() {
     drive.setDefaultCommand(
         DriveCommands.joystickDrive(
             drive,
-            () -> OI.getAxisSupplier(usingKeyboard ? OI.Keyboard.AD : OI.Driver.LeftY).get(),
+            () -> -OI.getAxisSupplier(usingKeyboard ? OI.Keyboard.AD : OI.Driver.LeftY).get(),
```

Comment:
This inversion works for the real robot, but breaks the keyboard controls. Not super important since it's only sim that's affected, but it would be nice to find a nice way of solving this.